website/static/js/logTextParser.js

```diff
@@ -92,7 +93,7 @@ var LogPieces = {
     node: {
         view: function (ctrl, logObject) {
             var nodeObject = logObject.embeds.nodes;
-            if(paramIsReturned(nodeObject, logObject)){
+            if(paramIsReturned(nodeObject, logObject) && nodeObject.data[0]){
                 return m('a', {href: nodeObject.data[0].links.html}, nodeObject.data[0].attributes.title);
```

Comment:
If I remember correctly, nodes is an array of the nodes involved in a nodeLog. Which means the first item will be the node the nodeLog belongs to, which is where all ` ${node} `